src/components/trade/index.tsx

```diff
@@ -91,6 +91,29 @@ export const TradeEntry = () => {
   return (
     <>
       <div className="input-card">
+        <Popover
+          placement="topRight"
```

Comment:
can you abstract that part so its one control that can be reused between swap, add window

Reply:
done